src/array.jl

```diff
@@ -193,18 +193,21 @@ beamform(s, f, sd; fs=framerate(s), method=Bartlett()) = beamform(s, f, 1, sd; f
 
 function beamformer(::Bartlett, R, sv)
   # TODO: add support for shading
-  [abs.(sv[:,j]' * R * sv[:,j]) for j ∈ 1:size(sv,2)]
+  RS = R * sv
+  pseudospectrum = zeros(Float64, size(sv, 2))
+  pseudospectrum .= abs.(dot.(eachcol(sv), eachcol(RS)))
```

Comment:
Would this work?
```suggestion
  abs.(dot.(eachcol(sv), eachcol(R * sv)))
```

Verification and benchmark:
```Julia
julia> sv = randn(64,256);
julia> R = let A = randn(64,64); (A + A')/2; end

julia> abs.(dot.(eachcol(sv), eachcol(R * sv))) ≈ [abs.(sv[:,j]' * R * sv[:,j]) for j ∈ 1:size(sv,2)]
true

julia> julia> @btime abs.(dot.(eachcol($sv), eachcol($R * $sv)));
  12.584 μs (6 allocations: 130.14 KiB)

julia> @btime [abs.($sv[:,j]' * $R * $sv[:,j]) for j ∈ 1:size($sv,2)];
  107.792 μs (1539 allocations: 434.06 KiB)
```